single-line-diagram-core/src/test/java/com/powsybl/sld/AbstractTestCase.java

```diff
@@ -76,7 +76,7 @@ String getName() {
 
     private void writeToFileInHomeDir(String filename, StringWriter content) {
         File homeFolder = new File(System.getProperty("user.home"));
-        File file = new File(homeFolder, filename);
+        File file = new File(homeFolder + File.separator + ".powsybl", filename);
```

Comment:
Why not? It sure is annoying to get all those files directly in your home folder. But you'd better do it in another PR.